base_comment_template/models/res_partner.py

```diff
@@ -9,6 +9,11 @@ class ResPartner(models.Model):
 
     base_comment_template_ids = fields.Many2many(
```

Comment:
I think you should include the field in commercial fields for parent/child propagation.